fermipy/scripts/flux_sensitivity.py

```diff
@@ -62,8 +62,16 @@ def main(args=None):
     parser.add_argument('--glat', default=0.0, type=float,
                         help='Galactic latitude in deg at which the sensitivity will be evaluated.  '
                         'Also sets the center of the sensitivity map for the `wcs` map type.')
+    parser.add_argument('--sedshape', default='PowerLaw', type=str,
+                        help='SED shape')
     parser.add_argument('--index', default=2.0, type=float,
```

Comment:
I'd put the allowed SED shapes into the help string explicitly.  Alternatively you could use the `choices` parameter in argparse:
```
parser.add_argument('--sedshape', default='PowerLaw', type=str,
           choices=['PowerLaw', 'PLExpCutoff', 'LogParabola'], help='SED shape')
```